cartography/models/aws/ec2/networkinterface_instance.py

```diff
@@ -0,0 +1,109 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class EC2NetworkInterfaceInstanceNodeProperties(CartographyNodeProperties):
+    """
+    Selection of properties of a network interface as known by an EC2 instance
+    """
+    # arn: PropertyRef = PropertyRef('Arn', extra_index=True) TODO use arn; issue #1024
+    id: PropertyRef = PropertyRef('NetworkInterfaceId')
+    status: PropertyRef = PropertyRef('Status')
+    mac_address: PropertyRef = PropertyRef('MacAddress')
+    description: PropertyRef = PropertyRef('Description')
+    private_dns_name: PropertyRef = PropertyRef('PrivateDnsName')
+    private_ip_address: PropertyRef = PropertyRef('PrivateIpAddress')
```

Comment:
But I think you meant to remove this `private_ip_address` form this particular schema?

Reply:
What you mean?

`private_ip_address` is present as a field on the node here: https://github.com/lyft/cartography/blob/830b8944879a01f52b21ee12b6fddf245f9733cb/cartography/intel/aws/ec2/network_interfaces.py#L51 so I'm making sure it's present in the refactor.

Should I remove it?

Reply:
From the PR description I thought you meant that the property only comes from `describe_network_interfaces`